src/lib/cli.ts

```diff
@@ -74,6 +71,11 @@ export class CliApplication extends Application {
                 if (this.json) {
                     this.generateJson(project, this.json);
                 }
+                if (!this.out && !this.json) {
+                    this.logger.log("You must either specify the 'out' or 'json' option.");
```

Comment:
If we are automatically setting a default "must" sounds strong here.
```suggestion
                    this.logger.log("No 'out' or 'json' option has been set");
```